pilot/pkg/features/pilot.go

```diff
@@ -403,6 +403,13 @@ var (
 		"If enabled, pilot will authorize XDS clients, to ensure they are acting only as namespaces they have permissions for.",
 	).Get()
 
+	TrustedCIDRRanges = env.RegisterStringVar(
```

Comment:
Let's add a comment here to avoid future confusion - this is temporary and experimental, will be replaced with Networks ( or something similar ).

Ok to get this in initially, and all env variables are considered internal/experimental - but this one in particular already has an existing API, and there are many reason to define it there.

Also, for name I would use TRUSTED_GATEWAY_CIDR - I don't think support for XFCC can be restricted to only PILOT, and RANGES seems redundant, but if you have to keep PILOT_ prefix I'm ok.
However I would rather qualify it - TRUSTED_CIDR can also mean networks where we don't need mTLS,
for pod to pod. 

"This can only be used when the network where Istiod and the authenticating gateways are running is a trusted/secure network"